Eatery/Controllers/BRB/BRBLoginViewController.swift

```diff
@@ -121,12 +80,13 @@ class BRBLoginViewController: UIViewController {
         setUpErrorView()
         setUpNetidViews()
         setUpPasswordViews()
-        setUpSaveLoginInfoViews()
         setUpLoginViews()
+    }
 
-        if let (netid, password) = BRBAccountSettings.loadFromKeychain(), !netid.isEmpty, !password.isEmpty {
-            netidTextField.text = netid
-            passwordTextField.text = password
+    override func viewWillAppear(_ animated: Bool) {
+        if !(netidTextField.text?.isEmpty ?? true) && !(passwordTextField.text?.isEmpty ?? true) {
```

Comment:
```
if let netID = netidTextField.text, 
  let password = passwordTextField.text, 
  !netID.isEmpty && !password.isEmpty {
  ...
}
```
might be a bit easier to read

Reply:
Wait, forgive me if I'm misreading this in my tired state, but aren't we setting the `netidTextField.text = nil` if it's not nil, and `passwordTextField.text = nil` if it's not nil? So is it possible to remove this `if` branch if necessary and just put `netidTextField.text = nil` and `passwordTextField.text = nil`?

Reply:
haha you're right, good catch